Tools/Create-Azure-Sentinel-Solution/input/Solution_CohesitySecurity.json

```diff
@@ -0,0 +1,21 @@
+{
+    "Name": "CohesitySecurity",
+    "Author": "Microsoft - support@microsoft.com",
+    "Description": "TBD",
+    "Data Connectors": [
+        "Data Connectors/helios_connector.json"
```

Comment:
@yinghuang123, there is no such file for data connector, can you please remove this, if the data connector won't be added in solution, thanks.

Reply:
The file was removed